examples/simple_key_value_store.rs

```diff
@@ -316,98 +320,94 @@ fn main() {
     // You can conveniently access values with `get_{bool,count,str,vec}`
     // functions. If the key doesn't exist (or if, e.g., you use `get_str` on
     // a switch), then a sensible default value is returned.
-    let node_type = args.get_str("--type");
     let mut command = String::new();
-    match node_type {
-        "node" => {
-            let mut test_node = RoutingNode::<TestNode, TestNodeGenerator>::new(TestNodeGenerator);
-            if !args.get_str("end_point").is_empty() {
-                match SocketAddr::from_str(args.get_str("end_point")) {
-                    Ok(addr) => {
-                        println!("initial bootstrapping to {} ", addr);
-                        let _ = test_node.bootstrap(Some(vec![Endpoint::Tcp(addr)]), None);
-                    }
-                    Err(_) => {}
-                };
-            } else {
-                // if no bootstrap endpoint provided, still need to call the bootstrap method to trigger default behaviour
-                let _ = test_node.bootstrap(None, None);
-            }
-            loop {
-                command.clear();
-                println!("Input command (stop)");
-                let _ = io::stdin().read_line(&mut command);
-                let v: Vec<&str> = command.split(' ').collect();
-                match v[0].trim() {
-                    "stop" => break,
-                    _ => println!("Invalid Option")
+    if args.get_bool("--node") {
+        let mut test_node = RoutingNode::<TestNode, TestNodeGenerator>::new(TestNodeGenerator);
+        if !args.get_str("peer").is_empty() {
```

Comment:
#324 doesn't cover it - this existing code doesn't even handle a single endpoint.

`args.get_str("peer")` will always be empty even if the user enters a peer endpoint since docopt sees `peer` as a vector.